index.html

```diff
@@ -116,7 +116,6 @@ <h2 class="project-title">Tonic</h2>
     </div>
 
     <div class="project-card">
-      <img src="./assets/Portoflio Card/multi_post_series.svg" alt="project card image" />
       <div class="prject-card-info flex column">
```

Comment:
- [ ] Behind the work section, I suggest making your background uniform from the headline section to the about section, as it will help us stick to the Figma design(For this kindly remove the white background behind the cards) :recycle: 

![Screenshot from 2023-02-23 12-07-44](https://user-images.githubusercontent.com/97834160/220877696-1f909063-2f9d-48db-8688-df155c36872c.png)

![Screenshot from 2023-02-23 12-10-10](https://user-images.githubusercontent.com/97834160/220877810-f25af407-248e-4f00-9ede-aa7d8085f4fc.png)